exchanges/stream/websocket.go

```diff
@@ -683,6 +679,31 @@ func (w *Websocket) FlushChannels() error {
 	return nil
 }
 
+// updateChannelSubscriptions subscribes or unsubscribes from channels and checks that the correct number of channels
+// have been subscribed to or unsubscribed from.
+func (w *Websocket) updateChannelSubscriptions(c Connection, store *subscription.Store, incoming subscription.List) error {
```

Comment:
Please add a dedicated test to this function

Reply:
[Done here](https://github.com/thrasher-corp/gocryptotrader/pull/1717/commits/759e225bba1e50cac9bcc6e592a218c336579d2d)